src/log_batch.rs

```diff
@@ -480,7 +479,7 @@ where
     W: EntryExt<E>,
 {
     pub items: Vec<LogItem<E>>,
-    items_size: usize,
+    approximate_size: usize,
```

Comment:
Let's not drop the `items_` prefix, you can rename it to `items_approximate_size` if you like.